docs/contributing.md

```diff
@@ -6,51 +6,53 @@ Contributions are welcome and encouraged! We ask only that all contributors foll
 
 Before opening a PR to contribute code you should check that your changes work by running the test suite locally.
 
+```{important}
+:name: dependencies
+We use [pixi](https://pixi.sh/latest/) to manage dependencies, which you'll want to install to get started.
+```
+
+Run tests with the `pixi run --environment test run-tests` command. Some tests require downloading files over the network.
+Use the `run-tests-no-network` task if you want to run tests faster or have no internet access:
+
 ```bash
-mamba env create -f ci/environment.yml
-mamba activate virtualizarr-tests
-pre-commit install
-# git checkout -b new-feature
-python -m pip install -e . --no-deps
-python -m pytest
+# Run all tests
+pixi run --environment test run-tests
+# Skip tests that require a network connection
+pixi run --environment test run-tests-no-network
```

Comment:
Why is it necessary to make running the tests a `pixi` thing, rather than just installing dependencies using pixi and then running tests using `pytest` directly?